examples/search-filter/byDate/js.md

```diff
@@ -1,3 +1,6 @@
-```
-
+```javascript
+const response = await fetch(
+  "https://dev.demo.sensenet.com/OData.svc/Root/Content/IT/Document_Library?$filter=CreationDate gt datetime'2019-03-26T03:55:00'",
+  { credentials: "include" }
+);
 ```
```

Comment:
This REST request is not working :(

Reply:
thanks for mentioning, I'll open a 🐜 issue for that